lib-ee/emqx_ee_connector/i18n/emqx_ee_connector_clickhouse.conf

```diff
@@ -0,0 +1,15 @@
+
+emqx_ee_connector_clickhouse {
+
+    base_url {
+        desc {
+          en: """The HTTP URL to the Clickhouse server that you want to connect to (for example http://myhostname:8123)"""
+          zh: """"""
```

Comment:
Just a reminder to fill these

Reply:
Fixed with auto translate. It should be checked by a Chinese speaker @JimMoen